apps/gains-agents/Dockerfile

```diff
@@ -54,8 +54,8 @@ COPY --from=builder /app/src /app/src
 COPY --from=builder /app/characters /app/characters
 COPY --from=builder /app/dist /app/dist
 COPY --from=builder /app/tsconfig.json /app/
-COPY --from=builder /app/pnpm-lock.yaml /app/
+#COPY --from=builder /app/pnpm-lock.yaml /app/
 
 EXPOSE 3000
 # Set the command to run the application
-CMD ["pnpm", "start", "--character=characters/eliza.character.json,characters/alon.character.json,characters/frank.character.json,characters/murad.character.json,characters/jesse.character.json", "--no-chat"]
\ No newline at end of file
+CMD ["pnpm", "start", "--character=characters/eliza.character.json,characters/alon.character.json,characters/frank.character.json,characters/murad.character.json,characters/jesse.character.json", "--non-interactive"]
```

Comment:
what is `non-interactive` does that work better than `no-chat`?

Reply:
looking at the discord chat and trying it out. It seems to be the same thing.